esp_encrypted_img/pre_encrypted_ota/main/idf_component.yml

```
@@ -0,0 +1,4 @@
+dependencies:
+  espressif/esp_encrypted_img: "^2.0.1"
```

```suggestion
  espressif/esp_encrypted_img:
      version: "^2.0.1"
      override_path: ../../../
```
otherwise when building this example from a git clone of idf-extra-components we will download another copy of esp_encrypted_img, instead of using the one here.